docs/release-notes.md

```diff
@@ -10,6 +10,11 @@ sidebarDepth: 1
 ---
 # Release Notes
 
+## 9.1.0
+### Bug Fixes and Improvements
+* Insights Agent 2.4.8
```

Comment:
Should this be a part of this release notes? Wasn't sure

Reply:
🤔 yeah maybe "Updated recommended Insights Agent version to 2.4.8"